librz/core/csyscall.c

```diff
@@ -0,0 +1,96 @@
+// SPDX-FileCopyrightText: 2009-2021 pancake <pancake@nopcode.org>
+// SPDX-FileCopyrightText: 2009-2021 maijin <maijin21@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_core.h>
+
+static const char *syscallNumber(int n) {
+	return sdb_fmt(n > 1000 ? "0x%x" : "%d", n);
+}
+
+/**
+ * \brief Returns the syscall representation as a string
+ *
+ * Given the syscall number and address it resolves the syscall
+ * for the selected `asm.arch` and `asm.os` values and print
+ * its arguments.
+ *
+ * \param core RzCore instance
+ * \param n number of the syscall
```

Comment:
there is no mention here about what this function does when `n` is `-1` or whether it is even valid (according to the code, it is valid). Please add doc for that so we don't have to read the function to know it's an expected value.